packages/test/test-end-to-end-tests/src/test/summarizeRefreshLatestSummary.spec.ts

```diff
@@ -53,5 +48,79 @@ describeCompat(
 			await createSummarizer(provider, container, undefined, summaryVersion);
 			await provider.ensureSynchronized();
 		});
+
+		itExpects(
+			"The summarizing client will immediately refresh its own summaries",
+			[
+				{ eventName: "fluid:telemetry:Summarizer:Running:Summarize_start" },
```

Comment:
These are not errors right? You should validate these via a mock logger to have more control over it. The itExpects block should be used for errors and not for test validations that are important.

Reply:
OK, will do